### ISSUE TITLE
Results Question

### ISSUE THREAD

**Issue by TheDreamerRandy** [HITSZ-HLT/JointCL#3]
How could I reproduce your results? I have tried your code with the same python libraries and hyper-parameters, but there is still a margin to your results published.

**Comment by BinLiang-NLP** [comment]
> How could I reproduce your results? I have tried your code with the same python libraries and hyper-parameters, but there is still a margin to your results published.

Sorry for the inconvenience. I'll check the code and reply to you asap. Many Thanks!

**Comment by minyaho** [comment]
Hi, I also have the same problem. I can't reproduce the result. I tried your code with the same version of python libraries and  hyperparameters which in the code, but the result was bad. I use the twitter_testDT_seenval dataset and the result is shown in the image below.

![image](https://user-images.githubusercontent.com/54090504/191235195-3179be05-2cac-4428-bfbd-c6f263b641ce.png)
![image](https://user-images.githubusercontent.com/54090504/191237042-28fea468-c3ba-4983-8af9-d944012aad42.png)

**Comment by Wenze7** [comment]
> > How could I reproduce your results? I have tried your code with the same python libraries and hyper-parameters, but there is still a margin to your results published.
> 
> Sorry for the inconvenience. I'll check the code and reply to you asap. Many Thanks!

Hi, i want to ask if your code has been checked?

**Comment by BinLiang-NLP** [comment]
> > > How could I reproduce your results? I have tried your code with the same python libraries and hyper-parameters, but there is still a margin to your results published.
> > 
> > 
> > Sorry for the inconvenience. I'll check the code and reply to you asap. Many Thanks!
> 
> Hi, i want to ask if your code has been checked?

Hi, I have checked the code, and I found that the results of some targets are OK, such as HC, LA, and A.
I am still checking the code of DT and CC to find the reason for the performance difference. Please wait a few days. Thanks!!!

**Comment by BinLiang-NLP** [comment]
> How could I reproduce your results? I have tried your code with the same python libraries and hyper-parameters, but there is still a margin to your results published.

Hi, sorry again for the inconvenience.
We have fixed the problems. Please run "git pull" to update the code.
We found that due to the small number of dataset samples (especially SEM16), the performance gap between different seeds will vary greatly. Thus, please tune the parameter "--seed" for better performance, no other parameter tuning is required.
Please let me know if there is any problem.
Thanks a lot!!!

**Comment by BinLiang-NLP** [comment]
> > > How could I reproduce your results? I have tried your code with the same python libraries and hyper-parameters, but there is still a margin to your results published.
> > 
> > 
> > Sorry for the inconvenience. I'll check the code and reply to you asap. Many Thanks!
> 
> Hi, i want to ask if your code has been checked?

Hi, sorry again for the inconvenience.
We have fixed the problems. Please run "_git pull_" to update the code.
We found that due to the small number of dataset samples (especially SEM16), the performance gap between different seeds will vary greatly. Thus, please tune the parameter "_--seed_" for better performance, no other parameter tuning is required.
Please let me know if there is any problem.
Thanks a lot!!!
